test/charts/utilization-bar/utilization-bar.spec.js

```diff
@@ -33,22 +33,27 @@ describe('Directive: pfUtilizationBarChart', function() {
     return el;
   };
 
-  it("should set the width of the inner bar to be 50%", function() {
+  it("should set the width of the inner bar to be 80%", function() {
     utilizationBar = angular.element(element).find('.progress-bar').css('width');
-    expect(utilizationBar).toBe("50%");
+    expect(utilizationBar).toBe("80%");
+  });
+
+  it("should set aria-valuenow values", function() {
+    used = angular.element(element).find('.progress-bar').not('.progress-bar-remaining');
+    expect(used.attr('aria-valuenow')).toBe("80");
   });
```

Comment:
With the above suggested changes, you should now be able to test for progress-bar-remaining values.